src/client/components/Fields.tsx

```diff
@@ -12,6 +12,40 @@ export type CustomFieldProps = WrappedFieldProps & {
   maxLength?: number;
 };
 
+type ApplicationColProps = {
+  className?: string;
+};
+
+type ApplicationLabelProps = {
+  required?: boolean;
+  className?: string;
+  forTag?: string;
+  // hack?
+  children?: string | JSX.Element[] | JSX.Element;
+};
+
+export type ApplicationInputProps = {
+  className?: string;
```

Comment:
Is it worth creating base types (then inheriting) if they have common properties? Such as className, required, placeholder, etc.? Many fields have these properties so it'd be nice if they aren't repeated.

Reply:
Oh good point! I forgot I was going to let the base inherit from JSX.InstrisicTypes (or whatever it's called) and pass the props down to allow things like aria tags to get on the fields

Reply:
ARIA fields 😮 Are we GDPR compliant as well? 😆

Reply:
Future proofing 🤣